app/brave_generated_resources.grd

```diff
@@ -963,6 +963,18 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
 
       <!-- Brave VPN -->
       <if expr="enable_brave_vpn">
+        <message name="IDS_SETTINGS_VPN_PAGE_TITLE" desc="Title of the vpn settings page">
+          Brave VPN
+        </message>
+        <message name="IDS_SETTINGS_VPN_PAGE_USE_WIREGUARD_TITLE" desc="Title of the Wireguard option on the vpn settings page">
+          Use WireGuard for Brave VPN
+        </message>
+        <message name="IDS_SETTINGS_VPN_PAGE_SUBLABEL_VPN_CONNECTED" desc="Title of the Wireguard sub label on the vpn settings page when brave vpn connected">
+          The mode cannot be changed while Brave VPN connected.
+        </message>
+        <message name="IDS_SETTINGS_VPN_PAGE_SUBLABEL_VPN_DISCONNECTED" desc="Title of the Wireguard sub label on the vpn settings page when brave vpn connected">
+          Use Wireguard protocol for Brave VPN, browser restart required.
```

Comment:
Suggestions: 
<table>
<tr>
<td>IDS_SETTINGS_VPN_PAGE_USE_WIREGUARD_TITLE</td><td>Use Wireguard protocol in Brave VPN</td>
</tr>
<tr>
<td>IDS_SETTINGS_VPN_PAGE_SUBLABEL_VPN_DISCONNECTED</td><td>Requires browser restart.</td>
</tr>
<tr>
<td>IDS_SETTINGS_VPN_PAGE_SUBLABEL_VPN_CONNECTED</td><td>This setting cannot be changed while Brave VPN is connected.</td>
</td>
</table>

cc: @rmcfadden3

Reply:
done, thanks